pkg/controller/bootstrap/bootstrap.go

```diff
@@ -40,6 +45,16 @@ func (b *Bootstrap) Run(destDir string) error {
 		return err
 	}
 
+	psfraw, err := ioutil.ReadFile(b.pullSecretFile)
```

Comment:
Will there always be a pull secret available, or will there be times pull secrets are defined and this will be an empty file?

Reply:
Its my plan to tighten to requirement when the source and location finalizes.